app/server.ts

```diff
@@ -135,6 +138,21 @@ async function initializeDatabase(): Promise<void> {
 }
 
 initializeDatabase().then(() => {
+  // Check that each JSON file in app/locales are valid JSON.
+  const localesPath = 'app/locales';
+  readdirSync(localesPath)
+    .filter((filename: string) => filename.endsWith('.json'))
+    .forEach((localeFilename: string) => {
+      const json = readFileSync(
+        path.join(localesPath, localeFilename),
+        'utf-8'
+      );
+      const { success, message } = checkIfValidLocaleJSON(localeFilename, json);
```

Comment:
Could just return message or null since we check on `message` in the if-statement, but no need to change it

Reply:
good idea, less clutter